step-templates/stackify-api-template

```diff
@@ -0,0 +1,121 @@
+      "Name": "app",
+      "Label": "Application Name",
+      "HelpText": "Name of your application. _Note this must match the name of an app in Retrace_",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      },
+      "Links": {}
+    },
+    {
+      "Id": "66237479-d4f0-48b7-82a6-01ea9659c45b",
+      "Name": "env",
+      "Label": "Environment",
+      "HelpText": "Name of your app's environment in Retrace. _Note this must match the application's environment in Retrace_",
+      "DefaultValue": "#{Octopus.Environment.Name}",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      },
+      "Links": {}
+    },
+    {
+      "Id": "a7d6da84-6b3f-460c-a8bf-7048304943d5",
+      "Name": "version",
+      "Label": "Version",
+      "HelpText": "What version of your app is this?",
+      "DefaultValue": "#{Octopus.Release.Number}",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      },
+      "Links": {}
+    },
+    {
+      "Id": "911474bd-527e-4b39-90e4-24ef861190e3",
+      "Name": "action",
+      "Label": "Deployment Action",
+      "HelpText": "either start/complete/cancel",
+      "DefaultValue": "complete",
+      "DisplaySettings": {
+        "Octopus.ControlType": "Select",
+        "Octopus.SelectOptions": "start|Start\ncomplete|Complete\ncancel|Cancel"
+      },
+      "Links": {}
+    },
+    {
+      "Id": "56dd8f21-65b8-4dbc-9247-cb5cbb80f86c",
+      "Name": "uri",
+      "Label": "Deployment URI",
+      "HelpText": "**OPTIONAL** A link you would like to associate with this release _e.g release notes or a link to this release_",
+      "DefaultValue": "#{Octopus.Web.ReleaseLink}",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      },
+      "Links": {}
+    },
+    {
+      "Id": "1fc3268f-7c4a-4877-8663-6fe00253af3c",
+      "Name": "branch",
+      "Label": "Branch",
+      "HelpText": "**OPTIONAL** _e.g. master_",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      },
+      "Links": {}
+    },
+    {
+      "Id": "5a4618a6-97a9-42f9-abd3-88234de9a8a1",
+      "Name": "commit",
+      "Label": "Commit",
+      "HelpText": "**OPTIONAL** A unique identifier to assign to this release.",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      },
+      "Links": {}
+    },
+    {
+      "Id": "645dab46-ee31-475b-a5a0-05eb059c0893",
+      "Name": "name",
+      "Label": "Release Name",
+      "HelpText": "**OPTIONAL** The name of this release.",
+      "DefaultValue": "#{Octopus.Deployment.Name}",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      },
+      "Links": {}
+    }
+  ],
+  "$Meta": {
+    "ExportedAt": "2018-01-29T22:12:56.674Z",
+    "OctopusVersion": "4.1.8",
+    "Type": "ActionTemplate"
+  }
```

Comment:
Missing `Category` property with the value `stackify`, would be good to also add a logo for `Stackify` as per our **contribution guide**

```
Optional: Assign your template to an existing category. Have a look at existing templates to find the category that matches your template. If you don't specify it your template will be assigned to 'other' category.
   - If you add a new category, make sure that you add an icon in `.png` format with a size of 200x200px to the `logos` folder with the same name as your category. Also, the `switch` in the `humanize` function in [`gulpfile.babel.js`](https://github.com/OctopusDeploy/Library/blob/master/gulpfile.babel.js#L92) must have a `case` statement corresponding to it.
```